.eslintrc

```diff
@@ -1,7 +1,8 @@
 {
   "env": {
     "node": true,
-    "jasmine": true
+    "jasmine": true,
+    "es6": true,
```

Comment:
I think this is being dealt with elsewhere?

Reply:
This was to stop it from bitching about consts and arrow funcs, sorry. We've already merged this change in, so it should be fine.